examples/manualrenderhint/src/index.js

```diff
@@ -153,7 +111,9 @@ const handleIsSwitchedOff = (trackState) => {
 
   // Disconnect from the Room
   window.onbeforeunload = () => {
-    stopVideoBitrateGraph();
+    if (stopVideoBitrateGraph) {
+      stopVideoBitrateGraph();
```

Comment:
```suggestion
      stopVideoBitrateGraph();
      stopVideoBitrateGraph = null;
```

Reply:
Here I'm getting an error: `stopVideoBitrateGraph` is not a function. This is because `stopVideoBitrateGraph()` is a promise, which returns a function to stop the bitrate graph. When I get this error, it prevents the participants from disconnecting from the room, so when I refresh this example a few times I see a `Too Many Participants in Room` error when the page loads. 

My question is, do we need to stop the bitrate graph when we close the browser or tab? Shouldn't closing the tab be enough to stop the graph?

Reply:
good find @timmydoza  about `stopVideoBitrateGraph` not being a function, I think we could make it function by this change:  https://github.com/twilio/video-quickstart-js/pull/185/files#r662521905